pkg/commands/git_commands/diff.go

```diff
@@ -14,6 +14,32 @@ func NewDiffCommands(gitCommon *GitCommon) *DiffCommands {
 
 func (self *DiffCommands) DiffCmdObj(diffArgs []string) oscommands.ICmdObj {
 	return self.cmd.New(
-		NewGitCmd("diff").Arg("--submodule", "--no-ext-diff", "--color").Arg(diffArgs...).ToArgv(),
+		NewGitCmd("diff").
+			Arg("--submodule", "--no-ext-diff", "--color").
+			Arg(diffArgs...).
+			ToArgv(),
```

Comment:
It took me a while of staring at this hunk to figure out that there's no change here other than formatting. I'd drop this to make future readers' lives easier.

Reply:
Yeah, I should have put that in a different commit, sorry for the confusion. I'll drop it.